DFC.Digital/DFC.Digital.Service.CourseSearch/Services/ConvertTribalCodesService.cs

```diff
@@ -0,0 +1,186 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace DFC.Digital.Service.CourseSearchProvider
+{
+    public class ConvertTribalCodesService : IConvertTribalCodes
+    {
+        public string[] GetTribalAttendanceModes(string attendanceMode)
+        {
+            if (string.IsNullOrWhiteSpace(attendanceMode))
+            {
+                return CourseSearchConstants.AllAttendanceModes.Split(',');
+            }
+
+            var selectedList = attendanceMode.Split(',');
+
+            if (selectedList.Contains("0"))
+            {
+                return CourseSearchConstants.AllAttendanceModes.Split(',');
+            }
+
+            var attendanceList = new List<string>();
+            foreach (var attendance in selectedList)
+            {
+                if (attendance.ToUpperInvariant().Equals("1"))
```

Comment:
Use switch case instead of multiple if else, check the cyclomatic complexity of this method.